test/index.test.js

```diff
@@ -84,7 +84,8 @@ for (const unitTest of fs.readdirSync(`${__dirname}/unit`)) {
 for (const cliTest of eval(fs.readFileSync(__dirname + "/cli.js").toString())) {
   it(`should execute "ncc ${(cliTest.args || []).join(" ")}"`, async () => {
     const ps = fork(__dirname + (coverage ? "/../src/cli.js" : "/../dist/ncc/cli.js"), cliTest.args || [], {
-      stdio: "pipe"
+      stdio: "pipe",
+      env: cliTest.env,
```

Comment:
Should this extend `process.env`?

Something like `{...process.env, ...cliTest.env}`